docs/design-v2.md

```diff
@@ -1,133 +1,77 @@
-# Azure Disk CSI Driver V2 (Project Mandalay)
+# Azure Disk CSI Driver V2
 
-Project Mandalay enhances the Azure Disk CSI Driver to improve scalability and reduce pod 
-failover latency. It uses shared disks to provision attachment replicas on multiple cluster 
-nodes and integrates with the pod scheduler to ensure a node with a attachment replica is 
-chosen on pod failover.
+Azure Disk CSI Driver V2 improves scalability and reduces pod failover latency. It uses shared disks to provision attachment replicas on multiple cluster nodes and integrates with the pod scheduler to ensure a node with an attachment replica is chosen on pod failover.
 
 ## Architecture and Components
 
-The diagram below shows the components in the Kubernetes control plane (CCP) and cluster 
-nodes, and the Azure services the Mandalay driver uses.
+The diagram below shows the components in the Kubernetes control plane (CCP) and cluster nodes, and the Azure services the Mandalay driver uses.
```

Comment:
@edreed Lets remove all references to Mandalay

Reply:
Agreed.